src/content/app/species/components/species-sidebar-modal/modal-views/SpeciesSidebarDownloads.tsx

```diff
@@ -26,53 +26,69 @@ import ExternalLink from 'src/shared/components/external-link/ExternalLink';
 import styles from './SpeciesSidebarDownloads.module.css';
 
 const SpeciesSidebarDownloads = () => {
+  //Fetch an array of ftp links and convert to object
   const activeGenomeId = useAppSelector(getActiveGenomeId) || '';
   const { data: fileLinksResponse } = useSpeciesFileLinksQuery(activeGenomeId, {
     skip: !activeGenomeId
   });
   const fileLinks = fileLinksResponse?.links || [];
-  const linkSections: Record<string, string> = {};
-
+  const fileLinkURLs: Record<string, string> = {};
   for (const link of fileLinks) {
-    linkSections[link.dataset_type] = link.path;
+    fileLinkURLs[link.dataset_type] = link.url;
   }
-
+  //Annotations from Andy
+  type LinkAnnotation = {
+    type: string;
+    title: string;
+    text: string;
+  };
+  type LinkSection = {
+    title: string;
+    links: LinkAnnotation[];
+  };
```

Comment:
please declare the types outside of the component (around line 27)

Reply:
Reverted

Reply:
Looks like it either didn't get reverted far enough, or got reverted too far 🙂 

<img width="820" alt="image" src="https://github.com/Ensembl/ensembl-client/assets/6834224/cb5765bb-5d88-47f8-83b2-5e4951cb32c7">